R/methods.R

```diff
@@ -93,14 +90,20 @@ method_bayes <- function(
     burn_between = 50,
     same_cov = TRUE,
     n_samples = 20,
-    seed = sample.int(.Machine$integer.max, 1)
+    seed = NULL
 ) {
+    if (!is.null(seed)) {
+        warning(paste0(
+            "The `seed` argument to `method_bayes()` has been deprecated",
+            " please use `set.seed()` instead"
+        ))
```

Comment:
```suggestion
        warning(
            "The `seed` argument to `method_bayes()` has been deprecated",
            " please use `set.seed()` instead"
        )
```
`paste0` is superfluous here, right?